src/ItspServices.pServer.Client/RestApi/RestApiClient.cs

```diff
@@ -1,21 +1,14 @@
-﻿using System.IO;
-using System.Net.Http;
+﻿using System;
 using System.Text.Json;
 using System.Threading.Tasks;
+using System.Net.Http;
 using ItspServices.pServer.Client.Models;
 
-[assembly: System.Runtime.CompilerServices.InternalsVisibleTo("ItspServices.pServer.ClientTest")]
-
 namespace ItspServices.pServer.Client.RestApi
 {
-    class RestApiClient
+    class RestApiClient : IApiClient
     {
-        IHttpClientFactory _provider;
-
-        public RestApiClient(IHttpClientFactory provider)
-        {
-            _provider = provider;
-        }
+        public IHttpClientFactory _provider { get; set; }
```

Comment:
If the IHttpClientFactory is a public property, then you should name it with UpperCamelCase notation ("Provider").